scripts/lsync.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
why add this?

Reply:
lol, this script is to find differences between en and zh since last commit😄